### PR TITLE
[544435] fixed issue with serializer and groups in non terminal rules

### PR DESCRIPTION
**Pull request by cdietrich**
[544435] fixed issue with serializer and groups in non terminal rules
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=544435

**Comment by cdietrich** [comment]
@meysholdt it would be nice if you could find some minutes to review. thanks

File: gradle/upstream-repositories.gradle [hunk 1]
```diff
@@ -24,6 +24,7 @@ def jenkinsPipelineRepo = { jobName, branch ->
 }
 
 repositories {
+  mavenCentral()
```
Comment (szarnekow) [inline gradle/upstream-repositories.gradle]: Necessary?

Comment (cdietrich) [reply gradle/upstream-repositories.gradle]: no this is the jcenter workaround

Comment (cdietrich) [reply gradle/upstream-repositories.gradle]: fixed

**Comment by cdietrich** [comment]
@szarnekow fixed the mavenCentral. is was there cause "jcenter crap"

**Pull request merged**